dist/profile/files/buildmaster/pipeline-configuration.groovy

```diff
@@ -21,6 +21,8 @@ LibraryConfiguration lib = new LibraryConfiguration('pipeline-library',
 
 lib.implicit = true
 lib.defaultVersion = 'master'
+lib.includeInChangesets = false
+lib.cachingConfiguration = new LibraryCachingConfiguration(180, null)
```

Comment:
relatively new feature thought we may as well try it out

Reply:
FTR, https://javadoc.jenkins.io/plugin/workflow-cps-global-lib/org/jenkinsci/plugins/workflow/libs/LibraryCachingConfiguration.html

Reply:
For info: https://github.com/jenkins-infra/helpdesk/issues/2751